ndcube/ndcube_sequence.py

```diff
@@ -239,6 +245,85 @@ def explode_along_axis(self, axis):
         # creating a new sequence with the result_cubes keeping the meta and common axis as axis
         return self._new_instance(result_cubes, common_axis=new_common_axis, meta=self.meta)
 
+    def _get_sequence_axes_wcs(self):
+        """
+        Returns a WCS object corresponding to the sequence axis of this NDCubeSequence.
+        It can contain multiple pixel axes that map to the same world axis.
+        """
+        table_coords = []
+
+        for axis_name, axis_coords in self.sequence_axis_coords.items():
+            if isinstance(axis_coords[0], (u.Quantity, numbers.Number)):
+                table_coords.append(QuantityTableCoordinate(u.Quantity(axis_coords),
+                                                            names=axis_name))
```

Comment:
In each case, try to pull the `physical_type` of each coord from the first cube in the sequence and use it to set the `physical_type` of each `TableCoordinate`.

Reply:
@adwaitbhope Any progress on this?

Reply:
I have now. I'm extracting the physical type from `global_coords`. But since we're iterating through `.items()` of the `sequence_axis_coords` dictionary, their order doesn't seem to be the same at every run.

I'm sorting the `wcs.world_axis_physical_types` in the tests due to this, is that alright?